model/labels/regexp_test.go

```diff
@@ -97,6 +98,13 @@ var (
 		"\xfd",
 		"\xff\xff",
 	}
+	toNormalisedLowerTestCases = map[string]string{
+		"foo":                      "foo",
+		"AAAAAAAAAAAAAAAAAAAAAAAA": "aaaaaaaaaaaaaaaaaaaaaaaa",
+		"cccccccccccccccccccccccC": "cccccccccccccccccccccccc",
+		"ſſſſſſſſſſſſſſſſſſſſſſſſS": "sssssssssssssssssssssssss",
+		"ſſAſſa": "ssassa",
+	}
```

Comment:
This is only used in `TestToNormalisedLower`, I would move the definition there to make the test easier to read & run.